### PR TITLE
feat: added new callback for card active status

### PR DESCRIPTION
**Pull request by moleinik**

### CODE REVIEW

File: src/components/SwipePan/index.tsx [hunk 1]
```diff
@@ -48,6 +50,7 @@ const SwipePan = ({
     onActive: (event, ctx) => {
       x.value = ctx.startX + event.translationX;
       y.value = ctx.startY + event.translationY;
+      runOnJS(onActive)(x, y);
```
Comment (tarasvakulka) [inline src/components/SwipePan/index.tsx:53]: Have you test onActive callback for performance issues? It's will generate a lot of call per second, at least we need to warn of a possible performance issue in the docs for this onSwipeActive method. Also would be great if you investigate some solutions to improve this.

Comment (moleinik) [reply src/components/SwipePan/index.tsx]: I updated this part. The new solution removes all unnecessary function calls. Check it

File: src/components/CardsSwipe/index.tsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ interface CardsSwipeProps {
   horizontalThreshold?: number;
   rotationAngle?: number;
   onSwipeStart?: (index: number) => void;
+  onChangeDirection?: (direction: SWIPE_DIRECTION) => void;
```
Comment (tarasvakulka) [inline src/components/CardsSwipe/index.tsx:43]: Rename pls to onSwipeChangeDirection

File: src/components/CardsSwipe/index.tsx [hunk 3]
```diff
@@ -122,6 +128,13 @@ const CardsSwipe = forwardRef(
       onSwipeStart(index);
     }, [index, onSwipeStart]);
 
+    const onActiveSwipe = useCallback(
```
Comment (tarasvakulka) [inline src/components/CardsSwipe/index.tsx:131]: Rename pls onActiveSwipe to onChangeSwipeDirection

File: README.md [hunk 4]
```diff
@@ -101,6 +101,7 @@ const styles = StyleSheet.create({
 | renderYep           | func          | renders Yep label                     |              | () => null    |
 | renderNope          | func          | renders Nope label                    |              | () => null    |
 | onSwipeStart        | func          | function to be called when a card swipe starts                  |    | () => {} |
+| onChangeDirection   | func          | function to be called when a card swipe in active status                  |    | () => {} |
```
Comment (tarasvakulka) [inline README.md:104]: Rename pls to onSwipeChangeDirection, update description and argument for this method

**Review by tarasvakulka — COMMENTED** [review]
Good job! New solution much more better. Update pls naming for new method and it will be ready to merge

File: src/components/CardsSwipe/index.tsx [hunk 5]
```diff
@@ -274,6 +283,7 @@ const CardsSwipe = forwardRef(
             {...{
               onSnap: onCardSwiped,
               onStart: onStartSwipe,
+              onActive: onChangeSwipeDirection,
```
Comment (tarasvakulka) [inline src/components/CardsSwipe/index.tsx:286]: onActive also rename pls to something like onChangeDirection

File: src/components/SwipePan/index.tsx [hunk 6]
```diff
@@ -33,10 +41,12 @@ const SwipePan = ({
   y,
   onSnap,
   onStart,
+  onActive,
```
Comment (tarasvakulka) [inline src/components/SwipePan/index.tsx:44]: onActive also rename pls to something like onChangeDirection

File: README.md [hunk 7]
```diff
@@ -101,6 +101,7 @@ const styles = StyleSheet.create({
 | renderYep           | func          | renders Yep label                     |              | () => null    |
 | renderNope          | func          | renders Nope label                    |              | () => null    |
 | onSwipeStart        | func          | function to be called when a card swipe starts                  |    | () => {} |
+| onChangeSwipeDirection | func          | function to be called when a card swipe in active status                  |    | () => {} |
```
Comment (tarasvakulka) [inline README.md:104]: name of the new method should be onSwipeChangeDirection, also update description to "function to be called when a card swipe change direction"

**Review by tarasvakulka — APPROVED** [review]
Looks good! Thank you a lot for your work

**Pull request merged**